### PR TITLE
Readme created.

### PR DESCRIPTION
**Pull request by vlobzakov**
JE-54257. Manifest, logo updated. Readme created.

### CODE REVIEW

File: mattermost.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 type: install
-name: Mattermost
+name: Mattermost Chat Service
 id: mattermost
 categories: ["apps/dev-and-admin-tools", "apps/popular"]
 displayName: Mattermost
-homepage: https://github.com/HidoraSwiss/mattermost
-logo: https://github.com/HidoraSwiss/mattermost/raw/master/mattermost.png
+homepage: https://mattermost.com/
+logo: https://github.com/vlobzakov/mattermost/raw/master/images/mattermost-logo-small.png
```
Comment (SlavaKatiukha) [inline mattermost.yaml]:
```suggestion
logo: https://github.com/jelastic-jps/mattermost/raw/master/images/mattermost-logo-small.png
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,94 @@
...
+<img src="images/addons.png" width="515"> 
+</p>
+
+Put custom domain into respective field. And press **Install**.
+
+<p align="left">
+<img src="images/ssl-addon.png" width="515"> 
+</p>
+
+  4. Open your application by new URL: **https://my-online-chat.jele.website** and make sure the application is workable and traffic is encrypted with valid SSL certificate issued for custom domain.
```
Comment (SlavaKatiukha) [inline README.md]: https://my-online-chat.jele.website
Do we need this link? It's too expensive to keep a separate environment for such cases + any user would be able to create an account there, which is not desirable.

**Review by SlavaKatiukha — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (vlobzakov) [reply README.md]: This is an example linkю Before this we have a phrase: For example let's create A record for custom domain **my-online-chat.jele.website**. 
So, what separate environment are you talking about?

Comment (SlavaKatiukha) [reply README.md]: who hosts **my-online-chat.jele.website**?

Comment (SlavaKatiukha) [inline README.md]:
```suggestion
  4. Open your application by new URL: `https://my-online-chat.jele.website` and make sure the application is workable and traffic is encrypted with valid SSL certificate issued for custom domain.
```

**Review by SlavaKatiukha — APPROVED** [review]

**Pull request merged**